cpuid/src/cpu_leaf.rs

```diff
@@ -3,6 +3,8 @@
 
 // Basic CPUID Information
 pub mod leaf_0x1 {
+    pub const FUNCTION: u32 = 0x1;
```

Comment:
I know `leaf` and `function` are used interchangeably everywhere, but to not confuse people that look at this code for the first time, I would say to stick with only one name: either leaf or function.

If we want to still call them leaves, than we can rename `FUNCTION` to `LEAF_NR`. What do you say?

P.S. Nice improvement from having the values of the leaves hardcoded.

Reply:
Done